test/acceptance/tests.py

```diff
@@ -66,9 +66,12 @@ class OpenAssessmentTest(WebAppTest):
         'file_upload':
             u'courses/{test_course_id}/courseware/'
             u'57a3f9d51d424f6cb922f0d69cba868d/bb563abc989340d8806920902f267ca3/'.format(test_course_id=TEST_COURSE_ID),
-        'full_workflow':
+        'full_workflow_override':
```

Comment:
Make this full_workflow_staff_override and full_workflow_staff_required so it is clear what the last word relates to.